pages/workspace/tracking-plan/implementation-status.mdx

```diff
@@ -75,3 +75,30 @@ We recommend using semantic versioning (e.g. "1.0.0") or other types of numeric
 
 Learn more about <Link passHref href="/implementation/avo-inspector-sdk-reference">how to install the Inspector in this doc</Link>.
 
+
+### Development and Staging
+
+  <img
+    src={require('../../../images/workspace/tracking-plan/dev-inspector-status.png')}
+    alt="Event never sent correctly with dev in the last 7d"
+    width="50%"
+  />
+
+The inspector implementation status can also report on how your tracking implementation is performing in dev and staging environments.
+
+However, because app version is rarely updated when working in dev environments, multiple versions of an event can exist between different branches
+that belong to the same app version. For that reason, instead of focusing on App version in dev and staging, each event is filtered by a time window (7d).
+
+> Time window will be editable in future release
+
+For each source we check the following for all events in your Tracking Plan:
+1. Have we seen the event on this source in the current timewindow provided?
+2. Is the event being sent according to the event definition in the current tracking plan?
+
+Based on these checks we report the implementation status using one of the following states:
+- Event is always sent correctly
+- Event is sometimes sent correctly
+- Event is never sent correctly
+
+
+Learn more about <Link passHref href="/implementation/start-using-avo-functions">how to start using Avo Functions in this doc</Link>.
```

Comment:
Shouldn't this be a call to action to start using the Inspector and move this call to action up to the Functions section?